src/actions/route_helpers.cr

```diff
@@ -15,6 +15,11 @@ module DppmRestApi::Actions
       {{method.id}} fmt_route(\{{route}}) do |\{{block.args.splat}}|
         namespace = \{{block.args[-1]}}.params.query["namespace"]? || DPPM::Prefix.default_group
         \{{block.body}}
+      rescue e
+        # Catch non-HTTPStatusError exceptions, and throw an InternalServerError
+        # with the original error as the cause.
+        raise e if e.is_a? HTTPStatusError
+        raise InternalServerError.new \{{block.args[0]}}, cause: e
```

Comment:
I think it would be clearer to do something like:
```cr
...
rescue ex : Actions
  raise ex
rescue ex
  InternalServerError
end
```
This means moving exception dedicated to `Actions`, in the `Actions` module.

Reply:
Are you suggesting that [this](https://github.com/DFabric/dppm-rest-api/blob/cfe755532ecbfb529208268df793ccd2b7482ef4/src/errors/http_status_errors.cr#L5) be renamed to `Actions` or `ActionException` or something? Or perhaps an alias like `alias Actions = HTTPStatusError | SomethingElse`?

Reply:
I submitted a commit to show what I was thinking: having `Actions::Exception`.